base/math.jl

```diff
@@ -1150,12 +1150,7 @@ for func in (:sin,:cos,:tan,:asin,:acos,:atan,:sinh,:cosh,:tanh,:asinh,:acosh,
     end
 end
 
-for func in (:atan,:hypot)
-    @eval begin
-        $func(a::Float16,b::Float16) = Float16($func(Float32(a),Float32(b)))
```

Comment:
Shouldn't we still use this for `hypot` of `Float16?

Reply:
We could, though as the PR was written, it would have been the wrong function to overload (since we want to always give it once chance to call `float`, then give it once chance to dispatch on the result, even if the type of the answer is the unchaged).
```
_hypot(a::Float16, b::Float16) = Float16(_hypot(Float32(a), Float32(b)))
_hypot(xs::Float16...) = Float16(_hypot(Float32.(xs)...))
```

Reply:
This indeed seems to have broken things https://github.com/JuliaLang/julia/issues/38311#issuecomment-722511170